docs/ref/contrib/gis/geos.txt

```diff
@@ -34,8 +34,7 @@ features include:
   may be used outside of a Django project/application.  In other words,
   no need to have :envvar:`DJANGO_SETTINGS_MODULE` set or use a database, etc.
 * Mutability: :class:`GEOSGeometry` objects may be modified.
-* Cross-platform and tested; compatible with Windows, Linux, Solaris, and
-  macOS platforms.
+* Cross-platform and tested.
```

Comment:
Wouldn't it be better to remove Solaris? I think the new version of that sentence hides information unnecessarily.

Reply:
We don't test on MacOS or Solaris, and on Windows testing is quite limited. IMO, there is no point in making false promises here.

Reply:
Got it. I didn't realize it's the case. Do we have a clean list of testing combinations?

Reply:
https://code.djangoproject.com/wiki/CI
djangoci.com